### PR TITLE
update: npm publish docs

### PR DESCRIPTION
**Pull request by ErinHales**
## Motivation

Published to npm, directions in `CONTRIBUTING.md` were out-of-date with what we've discussed in [slack](https://nomad-xyz.slack.com/archives/C02TSDESV0A/p1647550925134439?thread_ts=1647543677.076249&cid=C02TSDESV0A)

## Solution

Update docs

## PR Checklist

- [ ] Added Tests
- [x] Updated Documentation
- [ ] Updated CHANGELOG.md for the appropriate package

### CODE REVIEW

**Review by prestwich — APPROVED** [review]

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -106,16 +106,16 @@ Yes this is a lot, we intend to automate it later :)
...
 - Ensure that your package has a sensible `.npmignore`
 - Ensure that the `main` and `types` keys in your package are set correctly
-- Bump the version number in `package.json` (consult a senior dev if unsure)
-- Move all `Unreleased` changes to a new header with the new version number
-- Run yarn to ensure the lockfile is up to date
+- Update package version (yarn <package-to-release> version <strategy>[patch | minor | major](https://semver.org/))
 - Commit the above changes
 - Create a PR
-- AFTER the PR is merged, checkout master and tag the commit
+- AFTER the PR is merged, checkout master and tag the commit (git tag -s <package-to-release>@<new-package-version> <commit>)
```
Comment (Imti) [inline CONTRIBUTING.md:118]: should be main branch here

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -106,16 +106,16 @@ Yes this is a lot, we intend to automate it later :)
...
+Before publishing any package to `npm` perform the following steps:
 
+- Run yarn to ensure the lockfile is up to date
 - Format, lint, test, and build your package using the standard scripts
 - Ensure that your package has a sensible `.npmignore`
 - Ensure that the `main` and `types` keys in your package are set correctly
-- Bump the version number in `package.json` (consult a senior dev if unsure)
-- Move all `Unreleased` changes to a new header with the new version number
-- Run yarn to ensure the lockfile is up to date
+- Update package version (yarn <package-to-release> version <strategy>[patch | minor | major](https://semver.org/))
```
Comment (Imti) [inline CONTRIBUTING.md:115]: Keep in mind that the version strategy command doesn't work with release candidate versions

File: packages/sdk/tests/index.test.ts [hunk 3]
```diff
@@ -84,7 +84,7 @@ describe('sdk', async () => {
       expect(replica).to.not.be.undefined;
       const replicaFor = context.getReplicaFor(2000, 3000);
       expect(replicaFor).to.not.be.undefined;
-    })
+    });
```
Comment (Imti) [inline packages/sdk/tests/index.test.ts:87]: 🥳

**Review by Imti — APPROVED** [review]
📝

File: CONTRIBUTING.md (hunk 2 above)
Comment (ErinHales) [reply CONTRIBUTING.md]: Good to know, updated

**Pull request merged**